.nais/prod-gcp.yml

```diff
@@ -16,13 +16,13 @@ spec:
     min: 2
     max: 2
   liveness:
-    path: "/api/internal/isAlive"
+    path: /internal/alive
     initialDelay: 30
     timeout: 1
     periodSeconds: 30
     failureThreshold: 5
   readiness:
-    path: "/api/internal/isReady"
+    path: /internal/ready
     initialDelay: 30
     timeout: 1
   accessPolicy:
```

Comment:
Under ingresser i denne ligger det to urler. Bruker man "https://pensjon-selvbetjening-opptjening-frontend.intern.nav.no" til noe? Den hæres veldig ut som noe som kun skal aksesseres internt, og det er vel det den nye veileder-varianten skal støtte. (Samme mønster er også i dev-gcp.yaml)

Reply:
Intern-urlene er nok overflødige ja. Beholder dem foreløpig, så kan vi etterhvert fjerne dem og se om ting fortsatt virker.